modules/ipi-modify-a-disconnected-registry-config-yaml.adoc

```diff
@@ -0,0 +1,68 @@
+// Module included in the following assemblies:
+
+// * installing-mirroring-creating-registry.adoc
+
+:_content-type: PROCEDURE
+[id="ipi-modify-a-disconnected-registry-config-yaml_{context}"]
+= Modify the `install-config.yaml` file to use the disconnected registry
+
+The `install-config.yaml` file must contain the disconnected registry node's certificate and registry information.
+
+.Procedure
+
+. Add the disconnected registry node's certificate to the `install-config.yaml` file. The certificate should follow the `"additionalTrustBundle: |"` line and be properly indented, usually by two spaces:
```

Comment:
This should be duplicated with `Add the additionalTrustBundle parameter and value ` in the following step.